### PR TITLE
Consume the completion trigger character API

### PR DESCRIPTION
**Pull request by rchande**
Also take a dependency on version 1.17 where it was added and request to trigger completion on space.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1360,7 +1358,16 @@
             "request": "attach",
             "processId": "${command:pickProcess}"
           }
-        ]
+        ],
+        "windows": {
```
Comment (rchande) [inline package.json]: I'm not sure why these got added. Presumably they can be removed?

Comment (DustinCampbell) [reply package.json]: Remove these. Otherwise, they'll break the debugger.

**Comment by rchande** [comment]
@DustinCampbell Right--I was going to ask you if you if we had a different branch that tracked the latest insiders build (or something).

**Comment by rchande** [comment]
Also cc @piotrpMSFT and @jonsequitur in case they're interested.

**Comment by DustinCampbell** [comment]
We don't have a different branch as we haven't been on the bleeding edge with VS Code APIs yet.

**Comment by DustinCampbell** [comment]
Fixes #146

**Comment by rchande** [comment]
Thanks @DustinCampbell

**Comment by DustinCampbell** [comment]
For some reason, CI is hung for this PR. So, I went ahead and built and ran tests on my machine.

**Pull request merged**